man/systemd-fstab-generator.xml

```diff
@@ -198,6 +198,15 @@
         automatically populate <filename>/etc</filename>, and also <filename>/var</filename> in case of
         <literal>systemd.volatile=yes</literal>.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>systemd.swap=</varname></term>
+
+        <listitem><para>Takes a boolean argument. Defaults to
```

Comment:
The code does not require a boolean argument. Please make the docs match with the code... See the description of systemd.dump_core in systemd(1) for an example wording.

Reply:
Done